### PR TITLE
Update APT URLs for Ubuntu

### PR DESCRIPTION
**Pull request by nook24**
Inside of the Ubuntu documentation, the APT URLs were wrong.

### CODE REVIEW

File: docs/deployment/linux/ubuntu.txt [hunk 1]
```diff
@@ -47,17 +47,12 @@ the CrateDB repository.
 
 .. _Apt: https://wiki.debian.org/Apt
 
-Download the CrateDB GPG key:
+Download and import the CrateDB GPG key:
 
 .. code-block:: sh
 
-   sh$ wget https://cdn.crate.io/downloads/apt/DEB-GPG-KEY-crate
+   sh$ wget -qO - https://cdn.crate.io/downloads/deb/DEB-GPG-KEY-crate | apt-key add -
```
Comment (chaudum) [inline docs/deployment/linux/ubuntu.txt]: Adding the apt key requires su privileges, so it needs to be `... | sudo apt-key add -`

Comment (nook24) [reply docs/deployment/linux/ubuntu.txt]: Roger, I will update this tomorrow

**Comment by nook24** [comment]
@chaudum I don't know where my CLA request stuck inside our company, but looks like you fixed it in the current version of the docs, so we you can close the PR.

**Comment by chaudum** [comment]
Nevertheless, we're happy for new contributions any time :)

**Pull request closed without merging**